roles/ceph-crash/templates/ceph-crash.service.j2

```diff
@@ -17,6 +17,7 @@ ExecStart=/usr/bin/{{ container_binary }} run --rm --name ceph-crash-%i \
 {% if container_binary == 'podman' %}
 -d --log-driver journald --conmon-pidfile /%t/%n-pid --cidfile /%t/%n-cid \
 {% endif %}
+--pids-limit=0 \
```

Comment:
```suggestion
--pids-limit={{ 0 if container_binary == 'podman' else -1 }} \
```
docker and podman don't have the same unlimited value.

Reply:
Changes applied but forgot to squash the commit.